ecs-init/version/version.go

```diff
@@ -18,7 +18,7 @@
 package version
 
 // Version is the version of the ecs-init
-var Version string = "unknown"
+var Version string = "1.17.1"
```

Comment:
I think we aren't changing this var anymore right? This is set at build time now (always was IIUC).

Reply:
It was. I changed it out of habit and the update version script (called during build) sets it. Updated the PR to remove it.